frontend/app/scripts/controllers/diagnosticController.js

```
@@ -70,16 +70,16 @@ class DiagnosticCtrl {
-          fileIsOk(location.protocol + '//' + url + '/diag/' + $scope.basename, callbackFileIsOk);
-        },
-        err => {
-          $scope.collecting = false;
-          changeBtnText('collector.errors.timeout');
+        mqttClient.addStickySubscription('/wb-diag-collect/artifact', function (msg) {
+          if (msg.payload) {
+            const data = JSON.parse(msg.payload)
+            $scope.path = data['fullname'];
+            $scope.basename = data['basename'];
+            var url = getUrl();
+            fileIsOk(location.protocol + '//' + url + '/diag/' + $scope.basename, callbackFileIsOk);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)